src/bucket/Bucket.cpp

```diff
@@ -6,31 +6,31 @@
 // ASIO is somewhat particular about when it gets included -- it wants to be the
 // first to include <windows.h> -- so we try to include it before everything
 // else.
-#include "util/asio.h"
```

Comment:
Hmm, I've used .clang-format from stellar-core. There is no SortIncludes there.
I'll check compilation of this branch on Windows.